src/doctors/doctor.controller.ts

```
@@ -1,41 +1,41 @@
-  Delete,
-  ForbiddenException,
-  Get,
-  Param,
-  ParseIntPipe,
-  Post,
-  Put,
-  Query,
-  Req,
-  UseGuards,
-  UsePipes,
+    Body,
```

ESLint

[error] Delete `··`

(prettier/prettier)